.storybook/preview.tsx

```diff
@@ -42,15 +42,20 @@ const globalStyles = css`
 export const decorators = isEmbeddingSDK
   ? [] // No decorators for Embedding SDK stories, as we want to simulate real use cases
   : [
-      renderStory => (
-        <EmotionCacheProvider>
-          <ThemeProvider>
-            <Global styles={globalStyles} />
-            <CssVariables />
-            {renderStory()}
-          </ThemeProvider>
-        </EmotionCacheProvider>
-      ),
+      renderStory => {
+        if (!document.body.classList.contains("mb-wrapper")) {
```

Comment:
This is needed to make storybook work correctly

Reply:
This looks hacky. I tried putting this in `.storybook/preview-body.html`

```html
<body class="mb-wrapper"></body>
```

and it worked as expected. Can you try that?

![Screenshot 2024-11-29 at 8 41 57 PM](https://github.com/user-attachments/assets/45a4f6ad-fc2f-4556-a648-1d61e21448b3)

Reply:
@WiNloSt I think that will also apply to the sdk stories though, I don't think that's good as we'd miss all the issues that happen when our components are rendered outside
Out of scope for this pr, but I wish we had an entire different storybook for the sdk

Reply:
Good point. Let's keep this as is.